tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -854,7 +854,9 @@ PyMethodDef variable_methods[] = {
   {"__isub__", (PyCFunction)(void(*)(void))TypeError_to_NotImplemented_<THPVariable_sub_>, METH_VARARGS | METH_KEYWORDS, NULL},
   {"__div__", (PyCFunction)(void(*)(void))TypeError_to_NotImplemented_<THPVariable_div>, METH_VARARGS | METH_KEYWORDS, NULL},
   {"__truediv__", (PyCFunction)(void(*)(void))TypeError_to_NotImplemented_<THPVariable_div>, METH_VARARGS | METH_KEYWORDS, NULL},
+  {"__floordiv__", (PyCFunction)(void(*)(void))TypeError_to_NotImplemented_<THPVariable_floor_divide>, METH_VARARGS | METH_KEYWORDS, NULL},
```

Comment:
do you know why we manually bind these but some things in native_functions are registered there?  Like __and__ for example.  Maybe it has to do with TypeError_to_NotImplemented_ but I don't really see why those wouldn't apply to native_functions.

Reply:
We don't. I made a note to follow-up.